### DISCUSSION TITLE
Import mode

### DISCUSSION THREAD

**Discussion by spc789** [damionx7/Zettel-Notes-Documentation#96] [Feature Requests]
Give the app a directory path and it will import al .md .org files it finds (recursevely or not) into the db under a repo

**Comment by damionx7** [comment]
Hey! This is what Zettel Notes is based on. To select your .md notes folder in internal storage, Open Menu > Repositories > Add. 

Read more about repositories here https://znotes.thedoc.eu.org/repository/.
